generators/languages/templates/src/main/webapp/i18n/tr/global.json.ejs

```diff
@@ -68,7 +68,7 @@
         "messages": {
             "info": {
                 "authenticated": {
-                    "prefix": "",
+                    "prefix": "If you want to ",
```

Comment:
Maybe it should be empty, can it be fixed by checks?

Reply:
ok I'll let you see